### PR TITLE
add GetScaleformStatus

### PR DESCRIPTION
**Pull request by BaziForYou**
- added function to get active and inactive scaleform handles

### CODE REVIEW

**Comment by XenoS-ITA** [comment]
However, this only gets the scaleforms loaded from that script, since the native.lua is loaded individually for each script

**Comment by BaziForYou** [comment]
yes I know but this is the only way can manage on the easy way to see which gfx of your script is exist and status for all info to see which shows and what it doing and which is not active

File: client/native.lua [hunk 1]
```diff
@@ -1340,6 +1340,20 @@ end
     end)
 
 --// N3d //--
+    function GetScaleformStatus()
+        local ActiveList = {}
+        local InactiveList = {}
+        for i = 1, 10 do
+            local handleName = "utility_lib_" .. i
+            if Utility.Cache.N3d[handleName] then
+                table.insert(ActiveList, {Name = handleName, Data = Utility.Cache.N3d[handleName]})
```
Comment (XenoS-ITA) [inline client/native.lua:1349]: Hint: put the names in camel cases, since i usually use this type of case for table keys

**Comment by XenoS-ITA** [comment]
if you can change commit messages following [commit conventions](https://www.conventionalcommits.org/en/v1.0.0-beta.2/) (if you want)

File: client/native.lua (hunk 1 above)
Comment (BaziForYou) [reply client/native.lua:1349]: I can't understand what you mean about camel cases you mean dictionary way like this as the answer?
```lua
{
    ["utility_lib_1"] = blablabla,
    ["utility_lib_2"] = blablabla,
    ["utility_lib_3"] = blablabla,
    ["utility_lib_4"] = blablabla,
    ["utility_lib_5"] = blablabla,
    ["utility_lib_6"] = blablabla,
    ["utility_lib_7"] = blablabla,
    ["utility_lib_8"] = blablabla,
    ["utility_lib_9"] = blablabla,
    ["utility_lib_10"] = blablabla,
}
```

**Comment by XenoS-ITA** [comment]
i fixed what i had flagged in the comment and also added the IsScaleformTaken function to check a single scaleform, now you just need to tell me whether you want to change the commit messages or not, so i merge

**Comment by BaziForYou** [comment]
no all god i just changed typo of variables and now nothing to change

**Comment by XenoS-ITA** [comment]
im updating the documentation

**Pull request merged**